cfme/tests/cloud/test_provisioning.py

```diff
@@ -100,6 +98,10 @@ def test_provision_from_template(request, setup_provider, provider, testing_inst
              num_sec=1000,
              delay=60,
              handle_exception=True)
+    print(current_version())
+    if provider.type == 'gce' and current_version() > "5.7":
```

Comment:
preemtible VMs Checkbox has just appeared in 5.7.0.3. we are not going to run tests again at previous minor releases. and "> '5.6'" also covers all minor and Z releases from 5.6